grow/pods/dependency.py

```diff
@@ -0,0 +1,74 @@
+"""Dependency graph for content references."""
+
+import os
+
+class Error(Exception):
+    pass
+
+
+class BadFieldsError(Error, ValueError):
+    pass
+
+
+class DependencyGraph(object):
+    def __init__(self):
+        self.reset()
+
+    def add_all(self, data):
```

Comment:
What format is `data`? The convention we've used typically for naming is something like `file_ids_to_docs` (if `data` is a mapping of file IDs to documents, for example).